selenium_webdriver/src/test/java/tests/sauceDemoTests/SauceDemoAddItemToCartTest.java

```diff
@@ -0,0 +1,24 @@
+package tests.sauceDemoTests;
+
+import models.User;
+import org.testng.annotations.Test;
+import pages.SauceDemoLoginPage;
+import core.service.UserCreator;
+import tests.BaseTest;
+
+import static org.testng.Assert.assertEquals;
+
+public class SauceDemoAddItemToCartTest extends BaseTest {
+
+    @Test
+    public void oneCanAddItemToCart() {
+        User testUser = UserCreator.getStandardUser();
+        String buttonText = new SauceDemoLoginPage(driver)
+                .openSauceDemoLoginPage()
+                .loginToSauceDemoMainPage(testUser)
+                .clickAddToCartButton()
+                .getSauceLabsBackPackItemRemoveFromCartButtonText();
+
+        assertEquals(buttonText, "REMOVE", "An item was not successfully added to the cart.");
```

Comment:
Really you check not an adding item to the cart, you validate to button text or specific button presence

Reply:
Fixed